tutorials/hello-kubernetes/node/app.js

```diff
@@ -62,7 +62,7 @@ app.post('/neworder', async (req, res) => {
         if (!response.ok) {
             throw "Failed to persist state.";
         }
-        console.log("Successfully persisted state.");
+        console.log("Successfully persisted state for Order ID: " + orderId);
```

Comment:
I assume validation of a single line is more resilient and that's why we're changing this.  Correct me if wrong.